test/unit/services/test-reduction-service.js

```diff
@@ -45,6 +47,33 @@ var referenceData = [
   }
 ]
 
+var baseReduction =
+  { id: 1,
+    workloadOwnerId: 507,
+    hours: 5,
+    reductionReasonId: 1,
+    categoryId: 1,
+    reasonShortName: 'Disability',
+    allowancePercentage: null,
+    reductionStartDate: '11 Sep 16',
+    reductionEndDate: '09 Sep 27',
+    notes: null,
+    status: 'ACTIVE'
+  }
+
+var activeReductions = [ baseReduction ]
+
+var archivedReductions = [
+  Object.assign({}, baseReduction, { id: 2, status: 'ARCHIVED' }),
+  Object.assign({}, baseReduction, { id: 3, status: 'ARCHIVED' })
+]
+
+var reductionsByStatus = {
+  activeReductions: activeReductions,
+  scheduledReductions: [],
```

Comment:
Why not include scheduled reductions too?

Reply:
The idea was to show arrays of size 0, 1, and greater than 1 being processed